locale/en_US/manager.po

```diff
@@ -840,6 +840,15 @@ msgstr ""
 msgid "manager.settings.statistics.sushiPlatform.sushiPlatformID.required"
 msgstr "Platform ID is required"
 
+msgid "manager.settings.statistics.publicSushiApi"
+msgstr "Access for COUNTER SUSHI Statistics API"
```

Comment:
> COUNTER SUSHI Statistics

Reply:
@NateWr, above this form field there is a field "Platform for COUNTER SUSHI Statistics". 
Shall I leave this one only called "COUNTER SUSHI Statistics" or still rather something more specific (with access, availability or so)?
Sorry for double checking :-)